src/proto/bundler/bundler.proto

```diff
@@ -5,15 +5,29 @@ import "types/types.proto";
 
 package bundler;
 
-message ChainIdResponse {
-    uint64 chain_id = 1;
+enum Mode {
+    AUTO = 0;
+    MANUAL = 1;
```

Comment:
If it is auto, it would make sense to pass also bundle interval value (e..g, 10 secs or 5 secs ...).